package.json

```diff
@@ -3,7 +3,7 @@
   "version": "6.2.0",
```

Comment:
Can you up this to `6.2.1` please @meyer?

Reply:
ah, sure thing